app/templates/events/view/tickets.hbs

```diff
@@ -0,0 +1,123 @@
+<div class = "ui container">
+  <div class =  "ui grid">
+    <div class = "row">
+      <div class = "four wide column">
+        <div class="ui vertical pointing menu">
+          <a class="active item">Tickets</a>
+          <a class="item">Orders</a>
+          <a class="item">Attendees</a>
+          <a class="item">Discount Codes</a>
+          <a class="item">Access Codes</a>
+          <a class="item">Add order</a>
+        </div>
+      </div>
+      <div class = "twelve wide column">
+        <div class = "ui segment">
+          <div class = "ui statistics">
+            <div class = "statistic"><div class = "value">4343</div><div class = "label">Orders</div></div>
+            <div class = "statistic"><div class = "value">0.00</div><div class = "label">Sales(US $)</div></div>
+            <div class = "statistic"><div class = "value">1</div><div class = "label">Tickets sold</div></div>
+        </div>
+      </div>
+      <br>
+    <div class = "row">
+      <h4>Order Summary&nbsp;<span>By order status</span></h4>
+      <table class = "ui table">
+        <thead>
+          <tr>
+            <th>Order Status</th>
+            <th>Total Tickets</th>
+            <th>Total Orders</th>
+            <th>Total Sales (US$)</th>
+            <th>View Reports</th>
+          </tr>
+          <tr>
+            <td>
+              <span class = "ui grey label">Cancelled</span>
+            </td>
+            <td>0</td>
+            <td>0</td>
+            <td>0</td>
+            <td>View orders / View attendees</td>
+          </tr>
+          <tr>
+            <td>
+              <span class = "ui yellow label">Pending</span>
+            </td>
+            <td>0</td>
+            <td>0</td>
+            <td>0</td>
+            <td>View orders / View attendees</td>
+          </tr>
+          <tr>
+            <td>
+              <span class = "ui red label">Expired</span>
+            </td>
+            <td>0</td>
+            <td>0</td>
+            <td>0</td>
+            <td>View orders / View attendees</td>
+          </tr>
+          <tr>
+            <td>
+              <span class = "ui blue label">Placed</span>
+            </td>
+            <td>0</td>
+            <td>0</td>
+            <td>0</td>
+            <td>View orders / View attendees</td>
+          </tr>
+          <tr>
+            <td><span class = "ui green label">Completed</span></td>
+            <td>0</td>
+            <td>0</td>
+            <td>0</td>
+            <td>View orders / View attendees</td>
+          </tr>
+        </thead>
+      </table>
+    </div>
+        <br>
+        <div class = "row">
+          <table class = "ui table">
+            <thead>
+              <th>Ticket Type</th>
+              <th>Ticket Capacity Set</th>
+              <th>
+                <span class = "ui green label">Completed orders</span>
```

Comment:
This shouldn't be a label. See the screenshot attached to the issue (#81). The whole cell needs to be colored. use semantic ui's table coloring classes to do it.